others/faq.md

```diff
@@ -5,45 +5,49 @@ Please check here first if you have any questions.
 
 ## Application
 
-### 1. macOS "Clash Nyanpasu" is damaged and can't be opened
+### 1. macOS `"Clash Nyanpasu" is damaged and can't be opened`
 
 Open **Terminal** and run:
 
 ```bash
 sudo xattr -r -d com.apple.quarantine /Applications/Clash\ Nyanpasu.app
 ```
 
-### 2. Where is the Clash Nyanpasu application / logs directory?
+### 2. Where is the Clash Nyanpasu Config/Data/Logs Directory?
 
 There are currently two ways to directly open it:
 
-- Right-click on the tray icon and choose **Open Dir - App Dir**. You can then open the application directory.
-- Enter the main program interface, go to the **Settings page**, find the **Open App Dir** and click the button to open the application directory.
+- Right-click on the tray icon and choose **Open Dir - Config Dir**. You can then open the config directory.
+- Enter the main app interface, go to the **Settings page**, find the **Open Config Dir** and click the button to open the config directory.
 
-> Clash Core directory and Logs directory are also accessible in the same way.
+> Clash Core directory, Data directory and Logs directory are also accessible in the same way.
 
 In situations where you cannot access the tray or enter the main interface, you can directly enter the path to open it. (The clash core directory is in the installation directory)
 
 #### Windows
 
 ::: info Tip
-Starting from v1.5.1, Clash Nyanpasu supports customizing the application directory on Windows. If you have used this feature, please replace the path accordingly.
+Starting from v1.5.1, Clash Nyanpasu supports customizing the app directory on Windows. If you have used this feature, please replace the path accordingly.
+Starting from v1.6.0, Clash Nyanpasu splits the app directory into a config directory and a data directory. You can choose to migrate the config directory on Windows.
 :::
 
-- Application directory: `C:/Users/Username/.config/clash-nyanpasu`
-- Logs directory: `C:/Users/Username/.config/clash-nyanpasu/logs`
+- App directory (before v1.6.0): `%USERPROFILE%\.config\clash-nyanpasu`
+
+- Config directory: `%APPDATA%\Clash Nyanpasu\config`
+- Data directory: `%LOCALAPPDATA%\Clash Nyanpasu\data`
+- Logs directory: `%LOCALAPPDATA%\Clash Nyanpasu\data\logs`
 
 #### macOS & Linux
 
-- Application directory: `$HOME/.config/clash-nyanpasu`
+- App directory: `$HOME/.config/clash-nyanpasu`
 - Logs directory: `$HOME/.config/clash-nyanpasu/logs`
```

Comment:
The application dir, aka core dir, is wrong. That should be the install dir. I suggest use a note instead.

Tips,

On Windows, the default install dir is under AppData/Local
On macOS, it should under Applications dir of a user
On Linux, it is a temp dir via a AppImage executable. Otherwise, it should be /usr/bin

Reply:
And, you can see common path at https://github.com/libnyanpasu/nyanpasu-utils/blob/main/dirs-utils/src/lib.rs